community/sc/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+./configure
+
+make
+make DESTDIR="$1" EXDIR="$1/usr/bin" LIBDIR="$1/usr/lib" MANDIR="$1/usr/share/man" PREFIX=/usr install
```

Comment:
```
make \
    DESTDIR="$1" \
    EXDIR="$1/usr/bin" \
    LIBDIR="$1/usr/lib" \
    MANDIR="$1/usr/share/man" \
    PREFIX=/usr \
    install
```

Reply:
Question. Why `$1/usr/lib`, etc etc? Doesn't `/usr/lib/` work as you set `DESTDIR`?

Reply:
Because without `$1` kiss have no permissions to create the directory.

Reply:
```
install: can't create '/usr/bin/sc': Permission denied
install: can't create '/usr/bin/scqref': Permission denied
make: *** [Makefile:497: /usr/bin/sc] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:500: /usr/bin/scqref] Error 1
```

Reply:
i have problems with this package and permissions to create directories, `./configure` have no option to set a `--prefix=/usr` and this way is solved.

Reply:
> Because without `$1` kiss have no permissions to create the directory.

Excuse me, not kiss, makefile have not permissions to create the directories.

Reply:
OK. Just make the change I proposed in the first comment. :+1: